### ISSUE TITLE
Use VSC plugin to recommend alt tag content for images

### ISSUE THREAD

**Issue by bbringhu** [AdobeDocs/adobe-markdown-authoring#71]
Many images are missing alt tag content, which lowers SEO. Is it possible for the plugin to run a script that uses AI to generate missing or incomplete alt text for images?
